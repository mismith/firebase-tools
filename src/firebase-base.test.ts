import FirebaseBase from './firebase-base';

it('provides all required keys', () => {
  expect(Object.keys(FirebaseBase.keys)).toMatchSnapshot();
});
it('converts a String to a Date', () => {
  const date1 = FirebaseBase.getDate();
  const date2 = FirebaseBase.getDate(new Date().toString());
  expect(date1 instanceof Date).toBe(true);
  expect(date2 instanceof Date).toBe(true);
});
it('converts a Date to a String', () => {
  const date1 = FirebaseBase.getDateString();
  const date2 = FirebaseBase.getDateString(new Date());
  expect(typeof date1).toBe('string');
  expect(typeof date2).toBe('string');
});
