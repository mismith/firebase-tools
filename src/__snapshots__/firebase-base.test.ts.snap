// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`provides all required keys 1`] = `
Array [
  "link",
  "email",
  "payload",
  "created",
  "createdBy",
  "expires",
  "accepted",
  "acceptedBy",
  "cancelled",
]
`;
